src/class_resolver/api.py

```diff
@@ -135,6 +136,65 @@ def classes(self) -> Set[Type[X]]:
         """Return the available classes."""
         return set(self.lookup_dict.values())
 
+    def ray_tune_search_space(self, kwargs_search_space: Optional[Mapping[str, Any]] = None):
+        """Return a search space for ray.tune.
+
+        ray.tune is a package for distributed hyperparameter optimization. The search space for this search is defined
+        as a (nested) dictionary, which can contain special values `tune.{choice,uniform,...}`. For these values, the
+        search algorithm will sample a specific configuration.
+
+        This method can be used to create a tune.choice sampler for the choices available to the resolver. By default,
+        this is equivalent to
+
+            ray.tune.choice(self.options)
```

Comment:
this should be a code block

Reply:
I added `.. code-block:: python` for you